docs/style.css

```diff
@@ -393,3 +395,15 @@ body.dark-mode #abandon-game:active {
     background-color: #FFEB3B;
     border-color: #e57373;
 }
+
+#viewport{
+    padding: none;
+    display: block;
+    position: absolute;
```

Comment:
Does this need to be `absolute`? If you're not moving it anywhere, it might be better to make it `relative`. `absolute` is usually best for elements you want to place somewhere specific within the parent. `relative` is good when you just want to make an element sized so that `absolute` works in descendants. That way, you don't get janky effects if some rule tries to set `left` or whatever.